src/module.ts

```diff
@@ -1,12 +1,17 @@
 import { addImportsDir, addServerHandler, createResolver, defineNuxtModule, useLogger } from '@nuxt/kit'
-import { CreateStorageOptions } from 'unstorage'
 import { defu } from 'defu'
+import { BuiltinDriverName, builtinDrivers } from 'unstorage'
 
 export type SameSiteOptions = 'lax' | 'strict' | 'none'
 export type SupportedSessionApiMethods = 'patch' | 'delete' | 'get' | 'post'
 
+interface StorageOptions {
+  driver: BuiltinDriverName,
+  options: object
```

Comment:
Object can be almost everything in javascript, eg array is also an object. Is it possible to specify it further or even extend this type based on the `driver` name? I assume that every driver has its own set of options, or are they of the same type?

I think something like [mapped types](https://www.typescriptlang.org/docs/handbook/2/mapped-types.html) might help here

Reply:
I found each of the drivers option types from unstorage, so its using those now.